lib/chat_web/live/main_live/index.html.heex

```diff
@@ -227,8 +227,25 @@
           <div id="contentContainer" class="from-green-400 to-blue-500 hidden sm:flex flex flex-col contentContainer">
             <%= unless @dialog do %>
             <% else %>
+            <div id="chatGallery" class=" hidden relative bg-black opacity-80">
+              <button phx-click={JS.add_class("hidden", to: "#chatGallery") |> JS.remove_class("hidden", to: "#dialogInput") |> JS.remove_class("md:hidden", to: "#chatRoomBar") |> JS.remove_class("hidden", to: "#chatContent")} class="text-white flex relative left-5 top-10"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="white" viewBox="0 0 24 24"><path d="M16.67 0l2.83 2.829-9.339 9.175 9.339 9.167-2.83 2.829-12.17-11.996z"/></svg>
```

Comment:
It looks like we should move it at same level as rooms and chats
This way we would not need to duplicate this  for rooms and chats separately